csrc/executor_kernel_arg.cpp

```diff
@@ -24,42 +310,94 @@ PrimDataType TensorArgAbstract::getSmallestIndexType() const {
     }
   }
   return PrimDataType::Int32;
+  return PrimDataType::Int32;
```

Comment:
I guess this is due to a merge. Wonder why clang-tidy didn't say anything.

Reply:
fixed